src/main/java/tc/oc/pgm/PGMImpl.java

```diff
@@ -265,6 +268,8 @@ public void onEnable() {
     MapLoader mapLoader = new MapLoader(this, logger, registry);
     mapLibrary = new MapLibrary(logger);
 
+    StandardMapTags.registerDefaults(logger);
```

Comment:
I'm not sure if this should be called directly from `onEnable()`. Static initializers in interfaces aren't possible.
Related: https://github.com/TheMolkaPL/PGM-1/blob/a2cecd355b3894b8d4e22f712716f503d197b428/src/main/java/tc/oc/pgm/maptag/StandardMapTags.java#L41